<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BGDiyalog</class>
 <widget class="QDialog" name="BGDiyalog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuGen</string>
  </property>
  <property name="toolTip">
   <string>Bina Genelleştirme Eklentisi</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Turkish" country="Turkey"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>İşlemleri başlat!</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_layer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>İşlenecek katman! Bina ön hazırlık işlemi yapılmış katmanlar seçilmeli!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Katman Seç</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Paramereler 1:1000-1:25 000 dönüşümü için uygundur!</string>
   </property>
   <property name="text">
    <string>Parametreler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>İşlemler</string>
   </property>
  </widget>
  <widget class="QLabel" name="bilgi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>241</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Başladı!</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridParam">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Min. Alan (eleme)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="da1">
      <property name="toolTip">
       <string>Hedef ölçekte en küçük alan (m2)</string>
      </property>
      <property name="text">
       <string>25</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Dörtgen Tolerans</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="TolDo">
      <property name="toolTip">
       <string>Dörtgene dönüşüm oranı (0.9-1 arası değerler seçilmeli)</string>
      </property>
      <property name="text">
       <string>0.95</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Min. Alan (nokta)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="da2">
      <property name="toolTip">
       <string>Hedef ölçekte nokta geometrisine dönüşecek alanlar (m2)</string>
      </property>
      <property name="text">
       <string>625</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Daire Tolerans</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="TolDa">
      <property name="toolTip">
       <string>Daireye dönüşüm oranı (0.9-1 arası değerler seçilmeli)</string>
      </property>
      <property name="text">
       <string>0.95</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min. Alan (genell.)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="da3">
      <property name="toolTip">
       <string>Kontur genelleştimesinde girinti çıkıntılar için alan ölçütü, kaynak ölçekte m2 </string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Tampon (m)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="dtamp">
      <property name="toolTip">
       <string>Birleştirme için tampon uzaklığı (m). Min kenarın yarısı alınması önerilir!</string>
      </property>
      <property name="text">
       <string>2.5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Min. Açı (derece)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="daci">
      <property name="toolTip">
       <string>Minimum açı (derece)</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Min. Kenar (m)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="dknr">
      <property name="toolTip">
       <string>Minimum kenar (m)</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>212</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid_islem">
    <item row="2" column="1">
     <widget class="QCheckBox" name="KongenBox">
      <property name="toolTip">
       <string>Birleştirme sonrası kontur genelleştirmesi</string>
      </property>
      <property name="text">
       <string>Genelleştir</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="SekilBox">
      <property name="toolTip">
       <string>Birleştirme öncesi dörtgenleştirme ve daireleştirme yap.</string>
      </property>
      <property name="text">
       <string>Şekil Yap (Dörtgen/Daire)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="BirlesBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Birleştir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textMsg">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>İşlem günlüğü, uygulama bittiğinde içerik sistem belleğine aktarılır!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesajlar</string>
   </property>
  </widget>
  <widget class="QLabel" name="bilgi_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Bina genelleştirmesi modülü</string>
   </property>
   <property name="text">
    <string>Bina Genelleştirme Modülü BuGen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BGDiyalog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BGDiyalog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
